internal/experiments/experiments_remote_execute_api.go

```diff
@@ -0,0 +1,205 @@
+package experiments
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/operantai/secops-chaos/internal/categories"
+	"github.com/operantai/secops-chaos/internal/executor"
+	"github.com/operantai/secops-chaos/internal/k8s"
+	"github.com/operantai/secops-chaos/internal/verifier"
+	"gopkg.in/yaml.v3"
+)
+
+type RemoteExecuteAPIExperimentConfig struct {
+	Metadata   ExperimentMetadata `yaml:"metadata"`
+	Parameters RemoteExecuteAPI   `yaml:"parameters"`
```

Comment:
If all RemoteExecuteAPI tests are going to follow the same parameter pattern why not put the `RemoteExecuteAPI` struct inside the executor package?

```go
type RemoteExecuteAPIExperimentConfig struct {
	Metadata   ExperimentMetadata `yaml:"metadata"`
	Parameters executor.RemoteExecuteAPI   `yaml:"parameters"`
}
```